### DISCUSSION TITLE
Cart bundle - expected behavior is unclear

### DISCUSSION THREAD

**Discussion by maloguertin** [Shopify/function-examples#200] [Feedback: Cart Transform API]
Hi, I installed the Bundles reference app and the JSON Schema link made me discover this repo.

I am interested in starting to develop with the cartTransform function but I'm trying to figure out what the app should be showing.

**1. I created a parent product called Shirt**

<img width="650" alt="Screenshot 2023-02-22 at 4 44 26 PM" src="https://user-images.githubusercontent.com/6407064/220766332-48e8bb0b-8b0b-4595-90f5-1f5f5b044ec3.png">

**2. I created the variants that should be in the bundle**
**Added this json to both the variants:**

<img width="535" alt="Screenshot 2023-02-22 at 4 45 53 PM" src="https://user-images.githubusercontent.com/6407064/220766592-d369a7be-c82b-4479-83c3-b184bd3118b3.png">

**Now when I add both child variants separately to my cart this is what I see:**
![Screenshot 2023-02-22 at 4 49 47 PM](https://user-images.githubusercontent.com/6407064/220767457-38cc663d-031d-455f-9d9d-2c14e8364f82.png)

**And the checkout page**
![Screenshot 2023-02-22 at 4 50 13 PM](https://user-images.githubusercontent.com/6407064/220767500-36a19dbd-b3a4-42a0-b7b1-471f7ee9fc0c.png)

- Should the cart display the parent variant with the correct adjusted price?
- Should the subtotal and total price be displayed accounting for the price adjustment?

**When I add the parent variant directly**

I first see the parent variant in the cart, but if I go to checkout and come back to the cart it always displays both variants. Is this normal?

Cheers!

**Comment by ignacio-chiazzo** [comment]
Hey! Thanks for the feedback. 

> Should the cart display the parent variant with the correct adjusted price?

Yes, it should. It's a bug on our side. Please share the store ID with us so we can look closer. 

> Should the subtotal and total price be displayed accounting for the price adjustment?
It should. 

We will look into that.

**Comment by maloguertin** [comment]
This is the store domain: test-bundles-malo.myshopify.com

**Comment by ignacio-chiazzo** [comment]
Hi! Sorry for the late reply. The issue was fixed a while ago and I replied to the issue and not the discussion. Feel free to open if it is happening again.
